server/src/main/java/io/unitycatalog/server/persist/utils/HibernateUtils.java

```diff
@@ -67,4 +62,32 @@ private static SessionFactory createSessionFactory() {
       throw new RuntimeException("Exception during creation of SessionFactory", e);
     }
   }
+
+  private static Configuration getH2Configuration() {
+    Configuration configuration = new Configuration();
+    configuration.setProperty("hibernate.connection.driver_class", "org.h2.Driver");
+
+    if ("test".equals(properties.getProperty("server.env"))) {
+      configuration.setProperty("hibernate.connection.url", "jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1");
+      configuration.setProperty("hibernate.hbm2ddl.auto", "create-drop");
+      LOGGER.debug("Hibernate configuration set for testing");
+    } else {
+      configuration.setProperty(
+          "hibernate.connection.url", "jdbc:h2:file:./etc/db/h2db;DB_CLOSE_DELAY=-1");
+      configuration.setProperty("hibernate.hbm2ddl.auto", "update");
+      LOGGER.debug("Hibernate configuration set for production");
+    }
+
+    return configuration;
+  }
+
+  private static Configuration getMysqlConfiguration() {
+    Configuration configuration = new Configuration();
+    configuration.setProperty("hibernate.connection.driver_class", "com.mysql.cj.jdbc.Driver");
+    configuration.setProperty("hibernate.connection.url", "jdbc:mysql://localhost:3306/ucdb");
```

Comment:
we should put the connection.url, user, password values in the server.properties file